Client/Helpers/SpotlightHelper.swift

```diff
@@ -47,6 +47,10 @@ class SpotlightHelper: NSObject {
     }
 
     func update(pageContent: [String: String], forURL url: NSURL) {
+        if AboutUtils.isAboutURL(url) || url.scheme == "about" {
```

Comment:
Why not check for `http` and `https` here to lock it down to URLs that the `NSUserActivity` supports? I think there are a number of schemes that we can open, including `data` and `javascript`.